askama_parser/src/node.rs

```diff
@@ -878,38 +878,85 @@ impl<'a> Import<'a> {
 #[derive(Debug, PartialEq)]
 pub struct Call<'a> {
     pub ws: Ws,
+    pub caller_args: Vec<&'a str>,
     pub scope: Option<&'a str>,
     pub name: &'a str,
     pub args: Vec<WithSpan<'a, Expr<'a>>>,
+    pub ws1: Ws,
+    pub nodes: Vec<Node<'a>>,
+    pub ws2: Ws,
 }
 
 impl<'a> Call<'a> {
     fn parse(i: &mut &'a str, s: &State<'_, '_>) -> ParseResult<'a, WithSpan<'a, Self>> {
-        let start = *i;
+        let start_s = *i;
+        let parameters = |i: &mut _| -> ParseResult<'_, Option<Vec<&str>>> {
+            let args = opt(preceded(
+                '(',
+                (
+                    opt(terminated(separated(0.., ws(identifier), ','), opt(','))),
```

Comment:
This code doesn't handle literals like strings or integers or even tuples, etc. Alo, for `caller`, no need to support arguments I suppose, however `super` can have named arguments iiuc. So maybe call `Expr::arguments` instead?

Reply:
what would the syntax look like you would only declare arguments to be received right? this is for the calle?

Reply:
Oh right, sorry I thought it was about the call, not the definition. Please ignore this comment then. ^^'